docs/source/tutorial/visualise_pipeline.md

```diff
@@ -97,11 +97,13 @@ kedro viz --load-file my_shareable_pipeline.json
 This will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
 
 
-## Visualise Plotly charts in Kedro-Viz
+## Visualising charts in Kedro-Viz
```

Comment:
```suggestion
## Visualise charts in Kedro-Viz
```
I tend to agree that "Visualising" sounds better here, but our style guide discourages the use of gerunds, especially in headings. When Jo (technical writer) is back she will be happy I said this anyway 😀